components/ReCaptchaComp.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { ReCaptcha } from 'react-recaptcha-google'
+
+class ReCaptchaComp extends Component {
+  constructor(props, context) {
+    super(props, context);
+    this.onLoadRecaptcha = this.onLoadRecaptcha.bind(this);
+    this.verifyCallback = this.verifyCallback.bind(this);
+  }
+  componentDidMount() {
+    if (this.captcha) {
+        console.log("started, just a second...")
+        this.captcha.reset();
+    }
+  }
+  onLoadRecaptcha() {
+      if (this.captcha) {
+          this.captcha.reset();
+      }
+  }
+  verifyCallback(recaptchaToken) {
+    // Here you will get the final recaptchaToken!!!  
+    console.log(recaptchaToken, "<= your recaptcha token")
+  }
+  render() {
+    return (
+      <div>
+        <ReCaptcha
+            ref={(el) => {this.captcha = el;}}
+            size="normal"
+            data-theme="dark"            
+            data-badge="inline"
+            render="explicit"
+            //Add the site key here 
```

Comment:
FYI it is able to run recaptcha on localhost. Just need to configure a few things on google's side.